packages/meditrak-server/src/kobo/manualKoBoSync.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { DataBroker } from '@tupaia/data-broker';
+import { respond } from '@tupaia/utils';
+import { syncWithKoBo } from './startSyncWithKoBo';
+
+export async function manualKoBoSync(req, res, next) {
+  const { models } = req;
+
+  const dataBroker = new DataBroker();
```

Comment:
Could probably have these two processes share a data broker, but with the model registry being a singleton there's not a lot of overhead to it 🤔